src/components/notification/daikin-notification.ts

```diff
@@ -202,7 +211,11 @@ export class DaikinNotification extends LitElement {
               html`<header class="text-[18px] font-bold flex-none">
                 ${this.title}
               </header>`}
-              <p class="text-[18px] flex-none">${this.description}</p>
+              <span part="description" class="text-[18px] flex-none">
```

Comment:
I think this should be `div`?

```suggestion
              <div part="description" class="text-[18px] flex-none">
```